spec/controllers/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a valid book suggestion' do
+      let!(:book_suggestion) { create(:book_suggestion) }
+      before do
+        post :create, params: { author: book_suggestion.author, title: book_suggestion.title,
+                                link: book_suggestion.link }
+      end
+      it 'creates the book suggestion' do
+        expect(BookSuggestion.last) != book_suggestion
+      end
+
+      it 'responds with 201 status' do
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'When creating an invalid book suggestion' do
+      let!(:book_suggestion) { create(:book_suggestion) }
+      before do
+        post :create, params: { author: book_suggestion.author, title: book_suggestion.title }
+      end
+      it 'doesn\'t create a new rent' do
+        expect(BookSuggestion.last) == book_suggestion
```

Comment:
Check the matchers for equality [here](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/equality-matchers)